### PR TITLE
Updated events and info on Msiexec parameters

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5046

### CODE REVIEW

File: devices/surface/surface-dock-firmware-update.md [hunk 1]
```diff
@@ -83,9 +86,9 @@ Successful completion of Surface Dock Firmware Update results in new registry ke
...
-| Windows Device Install log       | %windir%\inf\ setupapi.dev.log         | For more information about using Device Install Log, refer [to SetupAPI Logging](https://docs.microsoft.com/windows-hardware/drivers/install/setupapi-logging--windows-vista-and-later-) documentation. |
+| -------------------------------- | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Surface Dock Firmware Update log | Path needs to be specified (Check note) | Earlier versions of this tool wrote events to Applications and Services Logs\Microsoft Surface Dock Updater.                                                                                                  |
+| Windows Device Install log       | %windir%\inf\ setupapi.dev.log          | For more information about using Device Install Log, refer [to SetupAPI Logging](https://docs.microsoft.com/windows-hardware/drivers/install/setupapi-logging--windows-vista-and-later-) documentation. |
```
Comment (illfated) [inline devices/surface/surface-dock-firmware-update.md]: Surplus space within the log filename path:
```suggestion
| Windows Device Install log       | %windir%\inf\setupapi.dev.log           | For more information about using Device Install Log, refer [to SetupAPI Logging](https://docs.microsoft.com/windows-hardware/drivers/install/setupapi-logging--windows-vista-and-later-) documentation. |
```

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by illfated** [comment]
To which extent does this PR touch on the contents of issue ticket #4984 (**Microsoft Surface Dock Updater Doesn't Exist**),
specifically the part that the updater software does not create any Start menu item or logs by default?
(just to draw a connection between these pages)

File: devices/surface/surface-dock-firmware-update.md [hunk 2]
```diff
@@ -83,9 +86,9 @@ Successful completion of Surface Dock Firmware Update results in new registry ke
...
-| Windows Device Install log       | %windir%\inf\ setupapi.dev.log         | For more information about using Device Install Log, refer [to SetupAPI Logging](https://docs.microsoft.com/windows-hardware/drivers/install/setupapi-logging--windows-vista-and-later-) documentation. |
+| -------------------------------- | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Surface Dock Firmware Update log | Path needs to be specified (Check note) | Earlier versions of this tool wrote events to Applications and Services Logs\Microsoft Surface Dock Updater.                                                                                                  |
+| Windows Device Install log       | %windir%\inf\setupapi.dev.log           | For more information about using Device Install Log, refer [to SetupAPI Logging](https://docs.microsoft.com/windows-hardware/drivers/install/setupapi-logging--windows-vista-and-later-) documentation. |
```
Comment (JohanFreelancer9) [inline devices/surface/surface-dock-firmware-update.md]:
```suggestion
| Windows Device Install log       | %windir%\inf\setupapi.dev.log           | For more information about using Device Install Log, refer to [SetupAPI Logging](https://docs.microsoft.com/windows-hardware/drivers/install/setupapi-logging--windows-vista-and-later-) documentation. |
```

File: devices/surface/surface-dock-firmware-update.md [hunk 3]
```diff
@@ -83,9 +86,9 @@ Successful completion of Surface Dock Firmware Update results in new registry ke
...
-| Surface Dock Firmware Update log | /l*v %windir%\logs\Applications\SurfaceDockFWI.log | Earlier versions of this tool wrote events to Applications and Services Logs\Microsoft Surface Dock Updater.                                                                                                  |
-| Windows Device Install log       | %windir%\inf\ setupapi.dev.log         | For more information about using Device Install Log, refer [to SetupAPI Logging](https://docs.microsoft.com/windows-hardware/drivers/install/setupapi-logging--windows-vista-and-later-) documentation. |
+| -------------------------------- | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Surface Dock Firmware Update log | Path needs to be specified (Check note) | Earlier versions of this tool wrote events to Applications and Services Logs\Microsoft Surface Dock Updater.                                                                                                  |
```
[the remark this answers is not in the document]
Comment (e0i) [reply devices/surface/surface-dock-firmware-update.md]: cc @VLG17

**Pull request merged**